src/legacy/core_plugins/telemetry/index.ts

```diff
@@ -23,6 +23,8 @@ import JoiNamespace from 'joi';
 import { Server } from 'hapi';
 import { CoreSetup, PluginInitializerContext } from 'src/core/server';
 import { i18n } from '@kbn/i18n';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { getConfigPath } from '../../../core/server/path';
```

Comment:
Should we actually expose this publicly from core/server? For this exemple, It seems like even once migrated to NP, the `config: Joi.string().default(getConfigPath()),` will still be required.

Reply:
I actually thought about that, but I was not entirely sure if we wanted to expose it publicly.
On the other hand, we are providing it in the plugin context so... why not :)
Changing it :)

Reply:
Actually, giving it a second thought. Not sure we should expose this publicly. It's risky since other plugins could read data outside their scope.

Reply:
they always can workaround it accessing the files directly. probably we should expose it. `telemetry` plugin uses this value

Reply:
Since comment https://github.com/elastic/kibana/pull/51478#discussion_r352020554, we are going to expose it in the context. Do we still need to expose it publicly? 🤔

Reply:
ah yes, via plugin context should be fine